Functions/RIPA.Functions.Stop/Functions/v1/GetStops.cs

```diff
@@ -63,40 +63,47 @@ public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Function, "
             log.LogError(ex.Message);
             return new UnauthorizedResult();
         }
-
+        List<IStop> stopResponse = new();
+        IEnumerable<StopStatusCount> stopStatusCounts;
         string stopV1QueryString = string.Empty;
         string stopV2QueryString = string.Empty;
         string stopSummaryQueryString = string.Empty;
-
+        int Limit = !string.IsNullOrWhiteSpace(req.Query["Limit"]) ? Convert.ToInt32(req.Query["Limit"]) : default;
         try
         {
             StopQueryUtility stopQueryUtility = new StopQueryUtility();
             StopQuery stopQuery = stopQueryUtility.GetStopQuery(req);
-            stopV1QueryString = stopQueryUtility.GetStopsQueryString(stopQuery, true, 1);
-            stopV2QueryString = stopQueryUtility.GetStopsQueryString(stopQuery, true, 2);
-            stopSummaryQueryString = stopQueryUtility.GetStopsSummaryQueryString(stopQuery);
+
```

Comment:
remove extra white space, one blank space between blocks before and after, and one blank space between methods.